### ISSUE TITLE
Invalid formatted text

### ISSUE THREAD

**Issue by kkeiichi** [EPPlusSoftware/EPPlus#1523]
### EPPlus usage

Personal use

### Environment

Windows 10

### Epplus version

7.2.1

### Spreadsheet application

Excel 2010

### Description

EPPlus does not return valid formatted text when the cell format contains a reference to "General". For example, in the attached file, the only cell has the format set to `General\ "mm"`. EPPlus instead of returning the formatted content `123 mm` returns `General mm`.

```csharp
using System;
using OfficeOpenXml;

namespace Test {
	internal class Program {
		static void Main(string[] args) {
			ExcelPackage.LicenseContext = LicenseContext.NonCommercial;

			using (var package = new ExcelPackage("example.xlsx")) {
				var sheet = package.Workbook.Worksheets[0];
				var text = sheet.Cells[1, 1].Text;
				Console.WriteLine(text);
			}
		}
	}
}

```

[example.xlsx](https://github.com/user-attachments/files/16159938/example.xlsx)

**Comment by JanKallman** [comment]
No this currently does not work. I will try to provide a fix.

**Comment by JanKallman** [comment]
Fixed in EPPlus 7.2.2
